src/clip.sol

```diff
@@ -91,14 +89,19 @@ contract Clipper is LibNote {
     uint256 public stopped = 0;
 
     // --- Events ---
+    event Rely(address indexed usr);
+    event Deny(address indexed usr);
+
+    event FileUint256(bytes32 indexed what, uint256 data);
```

Comment:
Why not calling both just `File`? The signature of the event is the name + the parameters so shouldn't be any problem to reuse the name.

Reply:
Just saw the comments from Kurt. Left my comment there.